flutter/lib/src/sentry_flutter_options.dart

```diff
@@ -199,6 +199,12 @@ class SentryFlutterOptions extends SentryOptions {
   /// Only available on iOS and macOS.
   bool enableAppHangTracking = true;
 
+  /// Connection timeout. This will only be synced to the Android native SDK.
+  Duration connectionTimeout = Duration(seconds: 5);
+
+  /// Read timeout. This will only be synced to the Android native SDK.
+  Duration readTimeout = Duration(seconds: 5);
```

Comment:
Are the default 5s for both of them on Android, right?

Reply:
Yeah, those values are taken form `sentry-java`. https://github.com/getsentry/sentry-java/blob/8ec2c16fc2919a60e1ac83845da19fdce6f71219/sentry/src/main/java/io/sentry/SentryOptions.java#L261